### PR TITLE
Update building-net-docker-images.md port mapping in Windows Container

### LINKED ISSUE #25708: Combine Run in a Windows container and Run in a Linux container
[Enter feedback here]

Note example in PR #25687
Related to issue #25708

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3653642e-61ef-717b-5436-9bd0a3c4e6f4
* Version Independent ID: 80c1ce51-46fb-0693-525a-7cad37150c7b
* Content: [Docker images for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-6.0&branch=pr-en-us-25687#run-in-a-windows-container)
* Content Source: [aspnetcore/host-and-deploy/docker/building-net-docker-images.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/host-and-deploy/docker/building-net-docker-images.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Jasonlhy**
Update port mapping in Windows Container, I run the instruction on my Windows 10 machine and it works

***@Rick-Anderson  EDIT*** 
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-6.0&branch=pr-en-us-25687#run-in-a-windows-container)
Fixes #25708

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25687) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25687) before we can accept your contribution.<br/><br/>:x: Jasonlhy [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25687)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=25687) it.</sub>

**Comment by Rick-Anderson** [comment]
@Jasonlhy  we can't look at your PR until you sign the CLA.
image

**Comment by Jasonlhy** [comment]
@Rick-Anderson I just signed the CLA

**Comment by Rick-Anderson** [comment]
@wadepickett  can you test this? 
@Jasonlhy can't we just delete the [Run in a Windows container](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-6.0&branch=pr-en-us-25687#run-in-a-windows-container) and change [Run in a Linux container](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-6.0&branch=pr-en-us-25687#run-in-a-linux-container) to ***Run in a Linux OR WINDOWS container** ?

**Comment by Jasonlhy** [comment]
@Rick-Anderson yes, I merged the instructions for Linux container and Windows container .

**Comment by Rick-Anderson** [comment]
@Jasonlhy thanks!
@wadepickett  let me know when you can test this.

**Comment by wadepickett** [comment]
@Rick-Anderson, sorry I missed your first ping.   
Tested running in a Windows container and tested build and deploy.  All worked fine.
Now testing with a Linux container...

**Comment by wadepickett** [comment]
... and running the Linux container, plus testing build and deploy all worked fine.  All instructions good.  Sample good.

File: aspnetcore/host-and-deploy/docker/building-net-docker-images.md [hunk 1]
```diff
@@ -120,9 +120,10 @@ The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.d
 
 * Press Ctrl+C at the command prompt to stop the app.
 
-## Run in a Linux container
+## Run in a Linux container or Windows container
 
-* In the Docker client, [switch to Linux containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
+* To run in a Linux container, in the Docker client, [switch to Linux containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
```
Comment (wadepickett) [inline aspnetcore/host-and-deploy/docker/building-net-docker-images.md:125]: There are two UI's involved here, the Docker app, and then the Docker client icon in the System Tray.   Making that clear for new Docker users so they don't spend time looking around the app UI for this.
```suggestion
* To run in a Linux container, right-click the System Tray's Docker client icon and select, [switch to Linux containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
```

File: aspnetcore/host-and-deploy/docker/building-net-docker-images.md [hunk 2]
```diff
@@ -120,9 +120,10 @@ The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.d
...
+* To run in a Windows container, in the Docker client, [switch to Windows containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
```
Comment (wadepickett) [inline aspnetcore/host-and-deploy/docker/building-net-docker-images.md:126]:
```suggestion
* To run in a Windows container, right-click the System Tray's Docker client icon and select [switch to Windows containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
```

**Review by wadepickett — APPROVED** [review]
Great work!

**Pull request merged**